### ISSUE TITLE
[Feature Request]Parallel download chapter

### ISSUE THREAD

**Issue by Rosabis** [tachiyomiorg/tachiyomi#5365]
## Why/User Benefit/User Problem
I wanted this feature where i can download multiple chapters at once in parallel.
## What/Requirements
(explain how this feature would behave)
Download at least 10 chapters at a time.

**Comment by loocool2** [comment]
Duplicate of #1567

**Comment by Rosabis** [comment]
> 
> 
> Duplicate of #1567

But the question is some sites will not cause IP bans if you are using parallel downloading

**Comment by CrepeTF** [comment]
Mass downloads put a lot of pressure on website's servers when Tachiyomi has over a million active users.

It's basically a non-intentional DDoS tool.

Parallel downloading for the same sources will not come.
